app/helpers/authentication_helper.rb

```diff
@@ -45,22 +45,22 @@ def waiting_on_first_user?
     Settings::General.waiting_on_first_user
   end
 
-  def private_forem_or_no_enabled_auth_options
-    ForemInstance.private? ||
+  def invite_only_mode_or_no_enabled_auth_options
+    ForemInstance.invitation_only? ||
       (authentication_enabled_providers.none? &&
        !Settings::Authentication.allow_email_password_registration)
   end
```

Comment:
Since we should be checking whether the particular forem instance is invite only or not within this method, I've taken the time to update the method's name here, and across the codebase, to help alleviate further confusion between invite only and private forems.